packages/editor/src/components/provider/disable-non-page-content-blocks.js

```diff
@@ -41,8 +45,10 @@ export default function DisableNonPageContentBlocks() {
 			for ( const clientId of contentOnlyIds ) {
 				setBlockEditingMode( clientId, 'contentOnly' );
 			}
-			for ( const clientId of templateParts ) {
-				setBlockEditingMode( clientId, 'contentOnly' );
+			if ( ! isNavigationMode ) {
+				for ( const clientId of templateParts ) {
+					setBlockEditingMode( clientId, 'contentOnly' );
+				}
```

Comment:
When editing a page (with 'Show template' on) in write mode, template parts are now not selectable.

Reply:
Oh, I was just wondering about this, and what `navigationMode` even means in this context.

So in this third useEffect, it sets template part blocks to content-only mode if not in navigation mode (which seems to be another term for write mode)?